ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -627,6 +635,15 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal, TwoLineTitl
         titleObserver = navigationItem.observe(\UINavigationItem.title) { [unowned self] item, _ in
             self.navigationItemDidUpdate(item)
         }
+        subtitleObserver = navigationItem.observe(\UINavigationItem.subtitle) { [unowned self] item, _ in
+            self.navigationItemDidUpdate(item)
+        }
+        titleAccessoryObserver = navigationItem.observe(\UINavigationItem.titleAccessory) { [unowned self] item, _ in
+            self.navigationItemDidUpdate(item)
+        }
+        titleImageObserver = navigationItem.observe(\UINavigationItem.titleImage) { [unowned self] item, _ in
```

Comment:
What are these observers used for or is that coming in a different PR?

Reply:
These were added because they correspond to the new fields we've added to UINavigationItem to be consistent with the other listeners. This was just something on my backlog, and I needed a PR to get them in with.